.eslintrc

```diff
@@ -49,6 +49,7 @@ rules:
   one-var: [2, never]
   prefer-arrow-callback: 0 # Wishlist, one day
   prefer-const: 0 # Wishlist, one day
+  prefer-template: 0 # Wishlist, one day
```

Comment:
Oh I understand now. I misread this line and thought we had added 2 and enabled it and that this PR had all the fixes.